### PR TITLE
Use Oleksiy's init scripts, which aren't broken

### PR DESCRIPTION
**Pull request by j0ni**
Closes appcanary/coalmine#227

Note that this doesn't make any changes for CentOS 7 which appears to have its own magical automation.

### CODE REVIEW

**Comment by j0ni** [comment]
*also need to test, which I will do on VMs and note results here.

**Comment by j0ni** [comment]
Tested on a CentOS 6 VM, installed the package, set up a watcher for `rpm -qa` and added my API key. Server was correctly started, and subsequent invocations of `/etc/init.d/appcanary start` did not result in more processes starting.

I believe this is good to go.

File: Rakefile [hunk 1]
```diff
@@ -67,7 +67,7 @@ task :cross_compile => :release_prep do
   puts "Cross compiling packages."
   puts "#################################\n\n\n"
 
-  @ldflags = %{-X main.CanaryVersion '#{@release_version}'}
+  @ldflags = %{-X main.CanaryVersion='#{@release_version}'}
```
Comment (mveytsman) [inline Rakefile]: was this broken the whole time?

Comment (j0ni) [inline Rakefile]: I think it broke after I upgraded to go1.8, which perhaps is more aggressive. But I'm not sure.

I think I also have a PR which switches to using `go` itself to cross-compile (appcanary/agent#152)

**Comment by j0ni** [comment]
@mveytsman so is that a :shipit: ?

**Pull request merged**